eth/stagedsync/stage_tevm.go

```diff
@@ -142,6 +145,13 @@ func transpileBatch(logPrefix string, s *StageState, fromBlock uint64, toBlock u
 		}
 	}
 
+	// Prune TEVM statuses if needed
+	if cfg.pruningDistance > 0 {
+		if err := pruneDupSortedBucket(tx, logPrefix, "TEVM statuses", dbutils.ContractTEVMCodeStatusBucket, toBlock, cfg.pruningDistance, logEvery.C); err != nil {
```

Comment:
`cfg.pruningDistance` in this case should just be replaced by `params.FullImmutabilityThreshold`